identity/client/src/pages/authorizations/List.tsx

```diff
@@ -46,6 +46,19 @@ const List: FC = () => {
     filter: { resourceType: activeTab },
   });
 
+  const sortPermissionTypesAlphabetically = (authorizationData: typeof data) =>
```

Comment:
Nit-pick: This is a static transformation function and does not need to be recreated with each render. Consider:

- moving the function definition outside of the render function
- Memoize the result and inline the transformation function in the useMemo call

Reply:
Thanks, Martin! I applied the suggestion.